packages/flame_svg/lib/svg.dart

```diff
@@ -96,8 +98,17 @@ class Svg {
 
   void _render(Canvas canvas, Size size) {
     canvas.scale(pixelRatio);
-    svgRoot.scaleCanvasToViewBox(canvas, size);
-    svgRoot.draw(canvas, svgRoot.viewport.viewBoxRect);
+
+    final scale = math.min(
+      size.width / pictureInfo.size.width,
+      size.height / pictureInfo.size.height,
+    );
+    canvas.translate(
+      (size.width - pictureInfo.size.width * scale) / 2,
+      (size.height - pictureInfo.size.height * scale) / 2,
+    );
+    canvas.scale(scale);
```

Comment:
sadly scaleCanvasToViewBox seems to no longer exist.
this could be vastly simpler with Vector2's, but that could introduce latency due to object allocation.

Reply:
dart really need inline classes/records/something like that